ci-operator/step-registry/sandboxed-containers-operator/env-cm/sandboxed-containers-operator-env-cm-ref.yaml

```diff
@@ -29,5 +29,14 @@ ref:
   - name: TEST_RELEASE_TYPE
     default: "GA"
     documentation: "The testing that is being carried out is for GA builds or Pre-GA builds"
+  - name: WORKLOAD_IMAGE
+    default: "quay.io/openshift/origin-hello-openshift"
```

Comment:
Just curious, what is the workload image used for? Is it the default image used when starting pods? Are there any requirements for it?

Reply:
Yes, it's the image used in the pod.
Coco needs a signed image and quay.io/openshift/origin-hello-openshift
 isn't signed.  We could use this variable.

What we are doing now though is using a different pod template with coco.  It has a cosigned image that is prebuilt.  And our peer-pod test that does a curl to it will fail because it's not running a web server